<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckIDDialog</class>
 <widget class="QDialog" name="CheckIDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckIDDialog</string>
  </property>
  <widget class="QPushButton" name="pBtnHomepage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>850</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/cancelon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnNext">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>850</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/nextoff.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>280</y>
     <width>750</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:20px;padding:2px 4px;
border-image: url(:/Resources/Images/sfzzmdw.png);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>312</y>
     <width>650</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/scanningid.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHead">
   <property name="geometry">
    <rect>
     <x>1537</x>
     <y>359</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoading">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>470</y>
     <width>124</width>
     <height>124</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/Images/loading.gif);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewScanningID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/doing.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewScanningVehicle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/prepared.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewInputFax">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/prepared.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewVehicleFax">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/undoneline.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewDone">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>135</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/prepared.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewFaxDone">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>150</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/undoneline.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScanningID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>扫描身份证</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScanningVehicle">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>扫描车辆信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInputFax">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>180</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>录入税费号</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDone">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>180</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewIDVehicle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/undoneline.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewIDSelect">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>1721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-image: url(:/Resources/Images/undoneline.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelect">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>180</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自选车牌</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="checkiddialog.qrc"/>
 </resources>
 <connections/>
</ui>
